terms/clinical/CERAD.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "sage.annotations-clinical.CERAD-0.0.1",
+    "description": "A semiquantitative measure of neuritic plaques",
+    "source": "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3266529/",
+    "anyOf": [
+        {
+            "const": 1
+        },
+        {
+            "const": 2
+        },
+        {
+            "const": 3
+        },
+	{
+            "const": 4
+        }
+    ]
+}
```

Comment:
Since we need a description and source for enumerated values, this might be better as an integer type with minimum 0, maximum 4.